### ISSUE TITLE
[Faceted Filters] Some UI not visible in high contrast

### ISSUE THREAD

**Issue by svinkle** [Shopify/dawn#112]
## Issue summary

The Sort By control and clear filters link border and 'x' icon are not visible in [Windows High Contrast mode](https://support.microsoft.com/en-us/windows/use-high-contrast-mode-in-windows-10-fedc744c-90ac-69df-aed5-c8a90125e696). In addition, facet control arrows are not visible in Edge. This makes it difficult to identify pieces of UI. Low-vision, high-contrast users should have equal access to content and imagery that sighted users have access to.

<details>
  <summary>Screenshots</summary>
<img width="1162" alt="Screen shot of faceted filters section with Windows High Contrast theme enabled. Controls are missing borders and icon identifiers." src="https://user-images.githubusercontent.com/1392632/124606960-67275600-de3b-11eb-8433-5dab3a5b956d.png">

</details>

#### HTML

```html
<svg
  aria-hidden="true"
  focusable="false"
  role="presentation"
  width="12"
  height="13"
  class="icon icon-close-small"
  viewBox="0 0 12 13"
  fill="none"
  xmlns="http://www.w3.org/2000/svg"
>
  <!-- … -->
</svg>
```

#### CSS

```css
.customer .field input,
.customer select,
.field__input,
.select__select {
  /* … */
  border: 0;
  /* … */
}

.button,
.customer button,
.shopify-challenge__button {
  /* … */
  border: 0;
  /* … */
}
```

## Steps to reproduce

1. Go to a collection page with Dawn theme with any supported browser.
2. With a Windows high contrast theme enabled, review the facet section styling.

### Expected

- Icons and controls to be visible in Windows High Contrast theme.

### Actual

- Icons and controls may not be easily distinguished.

## Recommendation

1. Add a `transparent` border for controls which are missing a visible border.
2. Adjust the `fill` attribute on the icon SVGs, setting its value to  to `currentColor`.

With these changes, Windows high contrast mode UI will be better suited for people with low-vision.

#### HTML

```html
<svg
  aria-hidden="true"
  focusable="false"
  role="presentation"
  width="12"
  height="13"
  class="icon icon-close-small"
  viewBox="0 0 12 13"
  fill="currentColor"
  xmlns="http://www.w3.org/2000/svg"
>
  <!-- … -->
</svg>
```

#### CSS

```css
.customer .field input,
.customer select,
.field__input,
.select__select {
  /* … */
  border: solid transparent 1px;
  /* … */
}

.button,
.customer button,
.shopify-challenge__button {
  /* … */
  border: solid transparent 1px;
  /* … */
}
```

<!-- #### Diff

```diff

``` -->

## Specifications

- **Component:** Faceted Filters
- **WCAG Principal:** Perceivable
- **WCAG SC:** [1.4.1 Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html)
- **Severity:** High
- **Effort:** Medium
<!-- - **Related:** -->

**Comment by gregjotau** [comment]
![Screenshot 2021-08-07 at 10 57 38](https://user-images.githubusercontent.com/13047497/128594915-3389015c-ef74-46cf-a795-7a368b3090d4.png)

I guess this is related: https://wave.webaim.org/report#/famme.no

**Comment by svinkle** [comment]
Fixed via #619.
